src/Microsoft.Azure.AppConfiguration.AspNetCore/AzureAppConfigurationRefreshMiddleware.cs

```diff
@@ -26,7 +26,7 @@ public async Task InvokeAsync(HttpContext context)
         {
             foreach (var refresher in Refreshers)
             {
-                _ = refresher.TryRefreshAsync();
+                _ = refresher.TryRefreshAsync(context.RequestAborted);
```

Comment:
This is an orphaned task. I don't think we want to require it to complete within the execution window of the request.

Reply:
Isn't `HttpContext.RequestAborted` equivalent to passing a CancellationToken? Do you mean we should check the `IsCancellationRequested` property before calling `TryRefreshAsync` with the cancellation token?